app/style/sass/common.scss

```diff
@@ -12,3 +13,25 @@
     display: inline-block;
   }
 }
+
+md-backdrop {
+  position: fixed;
+}
+
+.pgs-mainContent {
+  margin-top: 64px;
+
+  md-icon.star-icon {
+    width: 40px;
+    height: 40px;
+    position: absolute;
+    top: 10px;
+    right: 10px;
+  }
+}
+
+@media (min-width: 959px) {
```

Comment:
I would create:
1) variables that will define breakpoints in application (like `$tablet-min`, `$tablet-max`, `$desktop-min` etc
2) mixin that will take a device parameter and use breakpoint variables to create styles, like for example:
```scss
.pgs-mainContent {
    @include breakpoint(desktop) {
        margin-left: 320px;
    }
}
```